app/internal/page/data.go

```diff
@@ -140,3 +150,45 @@ func DecodeAddress(s string) *place.Address {
 	json.Unmarshal([]byte(s), &v)
 	return &v
 }
+
+func (l *Lpa) GetAttorney(id string) (Attorney, bool) {
```

Comment:
Agree with just returning bool for `PutAttorney()` but struggling to understand the changes to `GetAttorney()` (or was this all directed at `PutAttorney()` 😅)

Reply:
Directed at both. I think it is clearer to always be working with the value or the pointer rather than letting Go switch between depending on how it is called